lib/vagrant-libvirt/plugin.rb

```diff
@@ -23,7 +23,7 @@ class Plugin < Vagrant.plugin('2')
         Config
       end
 
-      provider('libvirt', parallel: true) do
+      provider('libvirt', parallel: true, box_optional: true) do
```

Comment:
Looks correct, commit is here: https://github.com/mitchellh/vagrant/commit/61ffa5313451c3705fe9e25c5d0167fae82fc865

1.5.0 will just error out if there is no box defined because it's not supported. I don't think we can print an error in vagrant-libvirt before vagrants validate runs. Will update the docs.